app/Kconfig

```diff
@@ -118,6 +118,14 @@ menuconfig ZMK_BLE
     select SETTINGS
     imply ZMK_BATTERY_REPORTING
 
+config ZMK_LOW_PRIORITY_THREAD_STACK_SIZE
+    int "Low priority thread stack size"
+    default 768
+
+config ZMK_LOW_PRIORITY_THREAD_PRIORITY
+    int "Low priority thread priority"
+    default 10
+
```

Comment:
This config is all reasonable, but I don't love *where* you're dropping it in this file. I would nest this under the "Advanced" menu somewhere.